paddle/fluid/operators/array_to_lod_tensor_op.cc

```diff
@@ -48,7 +48,8 @@ struct ArrayToLoDFunctor : public boost::static_visitor<void> {
 #ifdef PADDLE_WITH_CUDA
       Apply(static_cast<platform::CUDADeviceContext *>(pool.Get(place)));
 #else
-      PADDLE_THROW("Fluid is not compiled with CUDA");
+      PADDLE_THROW(
+          platform::errors::Unavailable("Fluid is not compiled with CUDA"));
```

Comment:
Fluid -> Paddle, we will remove fluid later

Reply:
Has changed to Paddle